### DISCUSSION TITLE
HCLOUD_TOKEN environment variable requirement

### DISCUSSION THREAD

**Discussion by PanadeEdu** [mysticaltech/terraform-hcloud-kube-hetzner#972] [Q&A] [answered]
I am currently trying to set this up for the fun of it. 
In my initial setup I was briefly stuck after executing the steps in:
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner#-do-not-skip-creating-your-kubetf-file-and-the-opensuse-microos-snapshot

Reason: 
The quick installation gives no output (at least for me), so i was not being made aware of the errors packer threw. 
Because of that my Terraform was not able to find the Images ofc. 

Never used packer before but looking into the Script that gets executed, I can see that HCLOUD_TOKEN is exportet before packer is executed. 
At no point the Docs stated this, and there is also no input for this token. 

Solution:
exported the HCLOUD_TOKEN manualy, ran packer commands again...works

**Comment by mysticaltech** [comment, marked as answer]
The `createkh` script handles that now, it will inform the user.
